src/extension.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as path from 'path';
+
+import { 
+	workspace, 
+	ExtensionContext, 
+	ConfigurationScope,
+	WorkspaceConfiguration
+ } from 'vscode';
+
+import {
+	LanguageClient,
+	LanguageClientOptions,
+} from 'vscode-languageclient/node';
+
+let client: LanguageClient;
+
+export function activate(context: ExtensionContext) {
+	const clientOptions: LanguageClientOptions = {
+		documentSelector: [{ scheme: 'file', language: 'terramate' }],
+		synchronize: {
+			// Notify the server about file changes to '.clientrc files contained in the workspace
+			fileEvents: workspace.createFileSystemWatcher('**/.clientrc')
+		}
+	};
+
+	client = new LanguageClient(
+		'terramate',
+		'terramate',
+		{
+			command: getServerPath(context),
+			args: getServerArgs(context),
+			options: { }
+		},
+		clientOptions
+	);
+
+	// Start the client. This will also launch the server
+	client.start();
+}
+
+export function deactivate(): Thenable<void> | undefined {
```

Comment:
![w](https://raw.githubusercontent.com/katcipis/memes/master/wat.jpg)

Reply:
Lazy naming is rampant in this community.